airbyte-integrations/connectors/destination-oracle/src/test-integration/java/io/airbyte/integrations/destination/oracle/NneOracleDestinationAcceptanceTest.java

```diff
@@ -48,7 +50,7 @@ public void testEncryption() throws SQLException {
     final List<JsonNode> collect = database.queryJsons(networkServiceBanner);
 
     assertThat(collect.get(2).get("NETWORK_SERVICE_BANNER").asText(),
-        equals("Oracle Advanced Security: " + algorithm + " encryption"));
+        is(equalTo("AES256 Encryption service adapter for Linux: Version 18.0.0.0.0 - Production")));
```

Comment:
why does this PR change `equals` assertion to `is(equalsTo())` ?

Reply:
`equals` was used not from the library for assertion, but from Object class, it means it did nothing previously